examples/compiled/bar_grouped_repeated.vg.json

```diff
@@ -52,7 +52,8 @@
           "y": [
             {
               "test": "!isValid(datum[\"sum_Worldwide Gross\"]) || !isFinite(+datum[\"sum_Worldwide Gross\"])",
-              "field": {"group": "height"}
+              "scale": "y",
```

Comment:
FYI, Good change as intended in a later fix commit -- invalid values get placed as zero rather than min.